### PR TITLE
osdocs8488rn: nodes release notes

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): 4.15

Issue: [OSDOCS-8488](https://issues.redhat.com//browse/OSDOCS-8488)

Link to docs preview: https://71920--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-nodes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by DCChadwick** [comment]
@tmalove Sorry about that! I put it up top and here, too: https://71920--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-nodes

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -707,6 +707,15 @@ For more information, see xref:../machine_management/control_plane_machine_manag
 
 [id="ocp-4-15-nodes"]
 === Nodes
+==== The `/dev/fuse` device enables faster builds on unprivileged pods
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:710]: No backticks in headings:
```suggestion
==== The /dev/fuse device enables faster builds on unprivileged pods
```

**Review by adellape — COMMENTED** [review]
Hold for title fix. Please re-queue for merge-review when addressed. Thank you!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:710]: Ideally also unique IDs added to the new headings. E.g.:
```suggestion
[id="ocp-4-15-nodes-dev-fuse"]
==== The /dev/fuse device enables faster builds on unprivileged pods
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -707,6 +707,15 @@ For more information, see xref:../machine_management/control_plane_machine_manag
...
+
+You can configure unprivileged pods with the `/dev/fuse` device to access faster builds.
+
+For more information, see xref:../nodes/containers/nodes-containers-dev-fuse.adoc#nodes-containers-dev-fuse[Accessing faster builds with /dev/fuse].
+
+==== Log linking is enabled default
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:716]: Unique ID:
```suggestion
[id="ocp-4-15-nodes-log-linking"]
==== Log linking is enabled default
```

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (DCChadwick) [reply release_notes/ocp-4-15-release-notes.adoc:710]: Thanks for catching these!

**Review by adellape — APPROVED** [review]

**Pull request merged**